lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -160,9 +160,7 @@ class FlutterViewEmbedder {
 
     // Create a [HostNode] under the glass pane element, and attach everything
     // there, instead of directly underneath the glass panel.
-    //
-    // TODO(dit): clean HostNode, https://github.com/flutter/flutter/issues/116204
-    final HostNode glassPaneElementHostNode = HostNode.create(
+    final HostNode glassPaneElementHostNode = HostNode(
```

Comment:
No need to keep `HostNode` at all, this should be a `DomShadowRoot`, IIRC.

Reply:
This involved moving many things around.. hopefully I didn't break the world lol 🤞